src/tcpreplay_opts.def

```diff
@@ -356,6 +356,19 @@ specify a maximum number of packets to send.
 EOText;
 };
 
+flag = {
+    name        = duration;
+    arg-type    = number;
+    max         = 1;
+    arg-default = -1;
+    arg-range   = "1->";
+    descrip     = "Limit the trasmit duration";
+    doc         = <<- EOText
```

Comment:
Typo. `trasmit` should be `transmit`.  Also, units are not specified in short help so possibly better to say `Limit the number of seconds to send`